app/uk/gov/hmrc/enrolmentsorchestrator/services/AuditService.scala

```diff
@@ -88,7 +87,7 @@ class AuditService @Inject() (auditConnector: AuditConnector)(implicit ec: Execu
   def auditClientDeleteResponse(arn: String, service: String, clientIdType: String, clientId: String, success: Boolean, statusCode: Int, failureReason: String)(implicit request: Request[_]): Unit = {
```

Comment:
sorry bit confused here, the method says it's for delete response but the audit type says delete request?